pages/tutorial/tutorial-series/tutorial_tutorial-series_part-2-event-listeners-and-template-listeners.md

```diff
@@ -122,7 +122,7 @@ The following event listeners achieves these requirements:
 Some notes on the code:
 
 - We are inheriting from `AbstractEventListener`, instead of just implementing the `IParameterizedEventListener` interface.
-  The `execute()` method of `AbstractEventListener` contains a dispatcher that automatically calls methods called `on` followed by the event name with the first character uppercased, passing the event object and the `$parameters` array.
+  The `execute()` method of `AbstractEventListener` contains a dispatcher that automatically calls methods called `on` followed by the event name with the first character uppercased (an event named `foo` is processed by the method `onFoo`), passing the event object and the `$parameters` array.
```

Comment:
```suggestion
  The `execute()` method of `AbstractEventListener` contains a dispatcher that automatically calls methods called `on` followed by the event name with the first character uppercased, passing the event object and the `$parameters` array. This simple pattern results in the event `foo` being forwarded to the method `onFoo($eventObj, $parameters)`.
```

Rationale: Inserting parts using paranthesis interrupt the flow. Having a dedicated sentence/example at the end acts as a short summary.